search/MakeTimingModel.cc

```diff
@@ -137,6 +138,9 @@ MakeTimingModel::makeLibrary()
 {
   library_ = network_->makeLibertyLibrary(lib_name_, filename_);
   LibertyLibrary *default_lib = network_->defaultLibertyLibrary();
+  if (default_lib == nullptr) {
+    criticalError(2141, "Cannot construct liberty model without first loading a liberty library [read_liberty]");
```

Comment:
criticalError exits OpenSTA. This shouild be an error.
I would prefer the wording to go more like
"No liberty libraries have been read."

The check should be in Sta::writeTiminigModel, as close to the command level as possible instead down in the trenches.

Reply:
What is the logger->error equivalent in OpenSTA?

Reply:
NVM. Looks like report_->error.